### PR TITLE
 Add a description meta tag 

### PR DESCRIPTION
**Pull request by garbas**
Continuation of #1016

### CODE REVIEW

**Comment by garbas** [comment]
@fricklerhandwerk @gilice Do you maybe have time to do a quick review?

File: layout.tt [hunk 1]
```diff
@@ -24,10 +24,23 @@ BLOCK navigationLink %]
...
+
 <html lang="en" class="without-js">
 
 <head>
-  <title>[% title %]</title>
+
+  [% IF title != "" %]
+  <title>[% title %] | Nix &amp; NixOS </title>
+  [% ELSE %]
+  <title>Nix &amp; NixOS | Reproducible builds and deployments</title>
```
Comment (fricklerhandwerk) [inline layout.tt:39]:
```suggestion
  <title>[% title %] | Nix </title>
  [% ELSE %]
  <title>Nix | Reproducible builds and deployments</title>
```

Except for the odd announcement, the current web site is exclusively about Nix and clearly not about NixOS. We have unambiguous evidence that beginners have a hard time distinguishing the different software components and their purpose and role. It of course needs more work to introduce the other projects and systems, but let's keep it focused on Nix for now.

**Review by fricklerhandwerk — APPROVED** [review]
I assume the code works, cannot test it right now.

File: layout.tt (hunk 1 above)
Comment (garbas) [reply layout.tt:39]: I'm leaving both in the title as we are also offering the download of NixOS.

And meanwhile we'll figure out the messaging on the website to introduce both Nix and NixOS ideas.

**Pull request merged**